widget/embedded/src/pages/HistoryPage.tsx

```diff
@@ -78,34 +78,27 @@ export function HistoryPage() {
       }}>
       {!loading && (
         <Container>
-          <TextField
-            prefix={<SearchIcon />}
+          <SearchInput
+            setValue={setSearchedFor}
             fullWidth
             color="light"
             variant="contained"
             placeholder="Search Transaction"
-            style={{
-              padding: 10,
-              borderRadius: 25,
-              alignItems: 'center',
-            }}
+            autoFocus
             onChange={filterTransactions}
             value={searchedFor}
           />
-          <SwapList>
-            {list.length ? (
-              <SwapsGroup
-                list={searchedFor ? filteredList : list}
-                onSwapClick={(requestId) => {
-                  setSelectedSwap(requestId);
-                  navigate(`${requestId}`, { replace: true });
-                }}
-                groupBy={groupSwapsByDate}
-              />
-            ) : (
-              <Alert type="info" title="Swap" />
-            )}
-          </SwapList>
+          <SwapsGroupContainer>
+            <SwapsGroup
+              list={searchedFor ? filteredList : list}
+              isSearched={!!searchedFor}
```

Comment:
I suggest don't consider a separate prop for this. if `list` prop is empty, we should show the `No results found`.